package.json

```diff
@@ -37,11 +37,10 @@
     "@babel/node": "^7.4.5",
     "@babel/preset-env": "^7.4.5",
     "@babel/register": "^7.4.4",
-    "chai": "^4.2.0",
-    "chai-http": "^4.3.0",
-    "dotenv": "^8.0.0",
+    "eslint": "^5.16.0",
```

Comment:
Confirm you didn't overwrite some packages required

Reply:
I did `git pull --rebase origin develop`, then there was a message that there's conflict that was not resolved in package-lock.json file so I accepted all incoming changes and did `git rebase --continue`. Thereafter, I did `git push origin my current branch` but I got an error message that my current branch is behind its remote counterpart so I did `git pull`. By recounting the process now, I realised that my last step must have overwritten the package.

I will make corrections.

Reply:
Next time you have an issue with package-lock.json. Just do ‘npm install’ and it will correct it

Reply:
Thank you, I just realised that now while troubleshooting the travis build error.

Reply:
So just do that to regenerate the lock file. That should resolve your Travis fail

Reply:
Done. Thank you